authentication/pipeline/compliance.py

```diff
@@ -55,7 +55,7 @@ def verify_exports_compliance(
                 mail.send_mail(
                     f"Exports Compliance: denied {user.email}",
                     f"User with email '{user.email}' was denied due to exports violation, for reason_code={export_inquiry.reason_code}, info_code={export_inquiry.info_code}",
```

Comment:
Is it possible to include the user's full name in this as well? I'm not sure, because the profile may not be populated yet.

Reply:
not sure either, if we really want to do then may be we can add a check before concatenating full name here. Please tell how/where we want to concatenate it.

Reply:
```suggestion
                    f"User with first name '{user.legal_address.first_name}', last name '{user.legal_address.last_name}, and email '{user.email}' was denied due to exports violation, for reason_code={export_inquiry.reason_code}, info_code={export_inquiry.info_code}",
```